scripts/slash-commands/reduce-risk3.sh

```
@@ -0,0 +1,270 @@
+# ✅ Step 1: Create Initial Snapshot
+echo -e "${BLUE}📸 Step 1: Creating initial snapshot...${NC}"
+INITIAL_SNAPSHOT=$(npm run --silent dev -- scan | grep -oP 'snapshot: \K[a-f0-9-]+')
+echo -e "${GREEN}✓ Initial snapshot: ${INITIAL_SNAPSHOT}${NC}"
+echo
+
+# 🏥 Step 2: Health-Guided Analysis
+echo -e "${BLUE}🏥 Step 2: Running health-guided analysis...${NC}"
+HEALTH_ANALYSIS=$(npm run --silent dev -- refactor health-analyze --format json --limit $MAX_FUNCTIONS --complexity-threshold $COMPLEXITY_THRESHOLD --priority-threshold $PRIORITY_THRESHOLD)
+
+# Extract high-priority functions
+HIGH_PRIORITY_FUNCTIONS=$(echo "$HEALTH_ANALYSIS" | jq -r '.plans[] | 
```

Shellcheck (0.10.0)

[warning] HIGH_PRIORITY_FUNCTIONS appears unused. Verify use (or export if used externally).

(SC2034)